lighthouse-core/scoring.js

```diff
@@ -6,6 +6,13 @@
 
 'use strict';
 
+/**
+ * Clamp figure to 2 decimal places
+ * @param {number} val
+ * @return {number}
+ */
+const clamp2decimals = val => Math.round(val * 100) / 100;
```

Comment:
Not the *biggest* fan of this name, clampTo2Decimals?

Reply:
👍